packages/dcos-diagnostics/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-diagnostics.git",
-    "ref": "20bc52c0e6521295f0332e4d91da98bd16c05903",
+    "ref": "99a19c1d7364d1464505ff9b3a363f40d173210c",
```

Comment:
This is actually the wrong hash. Current master is at `84f0d569086622d9c06c4f988100c21c5b474cd3` while `99a19c1d7364d1464505ff9b3a363f40d173210c` references a feature branch:
```
$ git branch -a --contains 99a19c1d7364d1464505ff9b3a363f40d173210c
  remotes/origin/allow_optional_commands
$
```

Reply:
Correct, I created this PR to check if tests are passing and they are not

Reply:
Ah, got it